zerver/lib/test_classes.py

```diff
@@ -1313,6 +1314,26 @@ def check_user_subscribed_only_to_streams(self, user_name: str, streams: List[St
         for x, y in zip(subscribed_streams, streams):
             self.assertEqual(x["name"], y.name)
 
+    def mark_topic_resolved(
+        self,
+        user: UserProfile,
+        msg_id_in_topic: int,
```

Comment:
I'm open to both the idea of renaming this method entirely to provide more clarity, renaming this variable to be less wordy since the docstring provides some further context here, or really any readability notes here. This feels reasonable to me in context, but "maybe not perfect?" enough that I felt like commenting on it.

Reply:
`target_message_id` might be a fine name? We use `target_` as a prefix for similar concepts.

(We also have a bit of a campaign to avoid using `msg_` as part of a general style rule against abbreviations and acronyms, as part of implementing our goal of making code harder to read for new contributors, though we have a lot of existing code that violates that pattern, so we can't add a linter rule for it)

Reply:
Love it (on both counts). Fixup'd and pushed.